WordPress/src/test/java/org/wordpress/android/ui/jetpack/restore/usecases/GetRestoreStatusUseCaseTest.kt

```diff
@@ -240,6 +241,14 @@ class GetRestoreStatusUseCaseTest {
                 )
             }
 
+    @Test
+    fun `given get status model is null, when restore status triggers, then return empty`() = test {
```

Comment:
Minor (🔍): Consider adding a test for the `without restore id` case. All the tests above are duplicated to `with restore id` and `without restore id`. Maybe we should keep that pattern for this test as well.

PS: After the so many update we did on this use case maybe the `with` and `without` restore id is no longer necessary. But, before deleting and consolidating these and the `Backup Download` related test, I would keep consistency first.
PSS: Feel free to ignore this comment.

Reply:
> PS: After the so many update we did on this use case maybe the `with` and `without` restore id is no longer necessary. 

I didn't think it was necessary, so I started on the road to eliminating it; however for this PR I will stay consistent. I added in the null restoreId test. :)

Reply:
Thank you @zwarm ! 🌟 

Yes indeed, I think it is no longer necessary and we can do an cleanup with our `Restore` and `Backup Download` use case tests next week when we will get the chance to do it.

PS: LGTM, feel free to merge this PR when it is ready.